demos/html5/froala/package.json

```diff
@@ -1,16 +1,19 @@
 {
   "name": "demo-html5-froala",
-  "version": "1.0.4",
+  "version": "1.1.0",
+  "private": true,
   "description": "A simple html5 App integrating WIRIS MathType in a Froala4 rich text editor.",
   "main": "app.js",
   "scripts": {
-    "compile-package": "cd ../../../packages/mathtype-froala3/ && npm run compile -- npm --dev",
+    "prestart-remote": "yarn && yarn unlink @wiris/mathtype-froala && yarn install --force",
+    "start-remote": "webpack-dev-server --open",
+    "prestart": "yarn && yarn link @wiris/mathtype-froala",
```

Comment:
I am unable to open the froala demo. 

Without removing the 3 from `@wiris/mathtype-froala` I am able to open the demo correctly.